pkgs/top-level/python-packages.nix

```
@@ -9056,6 +9056,8 @@ self: super: with self; {
 
   nextdns = callPackage ../development/python-modules/nextdns { };
 
+  nf-core = callPackage ../developement/python-modules/nf-core { };
```

Typo causing evaluation failure:

```suggestion
  nf-core = callPackage ../development/python-modules/nf-core { };
```

Tip: always copy paste such repetitive lines...